server/main.py

```diff
@@ -6,3 +6,7 @@
 @app.get("/")
 async def root():
     return {"message": "Hello World"}
+
+@app.get("/currency")
+async def root():
+    return{"message": "Price conversion"}
```

Comment:
For now I think we can just remove this. I'll create another ticket for the required currency related util functions